specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2023-10-01/redisenterprise.json

```diff
@@ -0,0 +1,2120 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-10-01",
```

Comment:
>version": "2023-10-01" [](http://example.com/codeflow?start=5&length=22)

I see this PR is adding new preview api version. preview version should have "-preview" suffix in api-version. Also indicate the same in PR template by selecting the checkbox that new api-version is getting introduced in the PR.

Reply:
Also, I have noticed the diff between latest commit to base commit does not shows all the new changes in the new api-version. Please fix it too.